src/main/cluster.ts

```diff
@@ -216,6 +216,18 @@ export class Cluster implements ClusterModel, ClusterState {
     return toJS({ prometheus, prometheusProvider });
   }
 
+  /**
+   * defaultNamespace preference
+   *
+   * @computed
+   * @internal
```

Comment:
internal not needed because `Cluster` is not anymore directly exposed to the extension api.